lib/pdk/module/templatedir.rb

```diff
@@ -236,26 +232,44 @@ def self.files_in_template(dirs)
       #
       # @api private
       def config_for(dest_path)
-        if @config.nil?
-          config_path = File.join(@path, 'config_defaults.yml')
+        sync_config_path = File.join(@path, '/.sync.yml')
```

Comment:
The `/` isn't necessary on a File.join. It will actually cause problems on Windows.